docs/FAQ/index.md

```diff
@@ -28,135 +36,138 @@ directory with product configuration and user groups in the Admin Console
 in order not to disturb our server every time for a list of users from the 
-org/product configuration?**<br/>
-A: No, User sync always queries the Adobe user management systems to get 
+org/product configuration?
+
+No, User sync always queries the Adobe user management systems to get 
 current information when it is run.
 
-**Q: I’d like to know more on how the user sync tool deals with the throttling 
-in general.**<br/>
-A: The Adobe systems protect themselves from overload by tracking the incoming 
+### I’d like to know more on how the user sync tool deals with the throttling 
+in general.
+
+The Adobe systems protect themselves from overload by tracking the incoming 
 request volume.  If this is starting to exceed limits, then requests return 
-a "retry-after" header indicating when capacity will be available again.  User sync honors these headers and waits for the requested amount of time before retrying.  More information, including code samples, can be found in the User Management API documentation at https://www.adobe.io/apis/cloudplatform/usermanagement/docs/throttling.html.
+a "retry-after" header indicating when capacity will be available again.  User sync honors these headers and waits for the requested amount of time before retrying.  More information, including code samples, can be found in the [User Management API documentation](https://www.adobe.io/apis/cloudplatform/usermanagement/docs/throttling.html).
  
-**Q: Is the user sync tool dedicated for federated Ids or any type of ID can be 
-created?**<br/>
-A: User sync supports all id types (Adobe IDs, Federated IDs and Enterprise IDs).
+### Is the user sync tool limited to federated Ids or can any type of ID can be 
+created?
 
-**Q: An org can have owned domains and domains to which another org has granted it 
+User sync supports all id types (Adobe IDs, Federated IDs and Enterprise IDs).
+
+### An org can have owned domains and domains to which another org has granted it 
 access. The UM API can run queries on both. How about user sync? Users are 
 added to the product configuration just because they are found in the AD group, 
-no check on the domain?**<br/>
-A: User Sync uses the UMAPI, so it can both query and manage the user 
+no check on the domain?
+
+User Sync uses the UMAPI, so it can both query and manage the user 
 groups and product access for users in both owned and accessed domains.  However, 
 like the Admin Console, User Sync can only be used to create and update user 
 accounts in owned domains, not domains owned by other organizations.
 
-**Q: Is there an update function, or just add/remove users (for only 
-federatedID)?**<br/>
-A: For all types of ID (Adobe, Enterprise, and Federated), User Sync supports 
+### Is there an update function, or just add/remove users (for only 
+federatedID)?
+
+For all types of ID (Adobe, Enterprise, and Federated), User Sync supports 
 update of group memberships under control of the --process-groups option.  
```

Comment:
This line, because it ends in spaces, is causing a spurious linebreak.  You need to make sure you don't have spaces at the ends of lines where the paragraph continues on the next line; two spaces at line end are interpreted as a `<br />` by Markdown.  See the preview.